extensions/wikia/DesignSystem/services/DesignSystemGlobalFooterService.class.php

```diff
@@ -2,6 +2,8 @@
 
 class DesignSystemGlobalFooterService extends WikiaService {
 	public function index() {
+		
+		
 		$this->setVal( 'model', $this->getData() );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "model" 13 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)